app/src/main/java/org/hackillinois/android/notifications/HackIllinoisNotificationManager.kt

```diff
@@ -0,0 +1,81 @@
+package org.hackillinois.android.notifications
+
+import android.app.*
+import android.content.Context
+import android.content.Intent
+import android.graphics.BitmapFactory
+import android.os.Build
+import org.hackillinois.android.R
+import org.hackillinois.android.database.entity.Event
+import java.util.*
+
+class HackIllinoisNotificationManager {
+    companion object {
```

Comment:
Do we need the `companion object`?

Reply:
No, I think we can just make HackIllinoisNotificationManager and object instead. Good call.